lib/Textmaster/Model/Project.php

```diff
@@ -207,6 +224,8 @@ public function setCallback(array $callback)
         }
 
         $this->data['callback'] = $callback;
+
+        return $this;
```

Comment:
Is this fluent interface used somewhere?
https://ocramius.github.io/blog/fluent-interfaces-are-evil/